src/modules/commentListener.js

```diff
@@ -1,57 +1,119 @@
-import getComments from './APIs/comments';
-
-const buildCommentsList = (comments, commentsList) => {
-  console.log(commentsList);
-  if (comments.length > 0) {
-    commentsList.innerHTML = comments
+import getComments from "./APIs/comments";
+const involvementURL =
+  "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/8ev0fUwJNQWCM4y1a4xa/";
+  
+  const buildCommentsList = (comments, commentsList) => {
+    if (comments.length > 0) {
+      commentsList.innerHTML = comments
       .map(
         (comment) => `
-          <li>
-            <p><em>${new Date(comment.creation_date).toLocaleDateString()} 
-            ${comment.username}</em> : ${comment.comment} </p>
-          </li>
-        `,
+        <li>
+        <p><em>${new Date(comment.creation_date).toLocaleDateString()} 
+        ${comment.username}</em> : ${comment.comment} </p>
+        </li>
+        `
       )
-      .join('');
+      .join("");
+    } else {
+      commentsList.innerHTML = "<li>No comments yet</li>";
+    }
+  };
+
+  const sendComment = async (pokemonID) => {
+    const commentForm = document.getElementById(`comment-form-${pokemonID}`);
+    const usernameInput = commentForm.querySelector(`#username-pk-${pokemonID}`);
+    const commentInput = commentForm.querySelector(`#comment-pk-${pokemonID}`);
+  const commentListComponent = document.querySelector(
+    `#comments-list-pk-${pokemonID}`
+  );
+  
+  const errorElement = document.getElementById(`error-${pokemonID}`);
+  console.log(errorElement);
```

Comment:
**[Optional]** Just wanted to suggest that you consider removing any unnecessary `console.log` statements from your code. 
> Doing so will help keep your code clean and easy to read! 😊